package.json

```diff
@@ -42,6 +42,7 @@
   "devDependencies": {
     "babel-cli": "6.11.4",
     "babel-preset-es2015": "6.6.0",
+    "babel-register": "^6.18.0",
```

Comment:
We should remove the `^` to set the version explicitely (instead of using a range of versions)